lib/s3-s3.js

```diff
@@ -83,8 +107,9 @@ var s3 = function (primaryS3, secondaryS3) {
      */
     getObject: function() {
       noArguments(arguments, 'getObject');
```

Comment:
You should just be able to very simply switch the secondary to the first position in the setup instead of the primary.  One I have the changes in dangerzone, this should be clearer.